### PR TITLE
feat: support different s3 storage class

### PR DESCRIPTION
**Pull request by brunopadz**
- Adds support for choosing S3 Storage Class #18 
- A bit of refactoring
- Updates documentantion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ S3 Cache for GitHub Actions supports builds on Linux, Windows and MacOS.
     aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
     aws-region: us-east-1 # Or whatever region your bucket was created
     bucket: your-bucket
+    s3class: STANDARD # It's STANDARD by default. It can be either STANDARD,
```
Comment (amendoa) [inline README.md:28]: Can you use `kebab-case`, please? 
About default value, you can use [default field](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs) for that.

File: action.yml [hunk 2]
```diff
@@ -25,6 +25,9 @@ inputs:
   artifacts:
     description: "A list of files, directories and glob patterns to cache and restore"
     required: false
+  s3class:
```
Comment (amendoa) [inline action.yml:28]: The same about default field and `kebab-case`

File: action.yml [hunk 3]
```diff
@@ -36,6 +39,7 @@ runs:
         AWS_SECRET_ACCESS_KEY: ${{ inputs.aws-secret-access-key }}
         AWS_REGION: ${{ inputs.aws-region }}
         BUCKET: ${{ inputs.bucket }}
+        S3_CLASS: ${{ inputs.s3class }}
```
Comment (amendoa) [inline action.yml:42]: `kebab-case` here

File: README.md (hunk 1 above)
Comment (brunopadz) [reply README.md:28]: I believe `camelCase` is a better approach since it's a matter of syntax standards.

**Review by amendoa — DISMISSED** [review]

**Review by amendoa — APPROVED** [review]

**Pull request merged**